scenarios/kubernetes_bazel.py

```diff
@@ -71,10 +71,15 @@ def main(args):
             res = 1
         else:
             try:
-                check(
-                    'bazel', 'run', '//:push-build', '--',
-                    '%s/%s' % (args.gcs, version)
-                    )
+                gcs_location = '%s/%s' % (args.gcs, version)
+                check('bazel', 'run', '//:push-build', '--', gcs_location)
+                artifacts = '%s/_artifacts' % os.environ['WORKSPACE']
```

Comment:
Any chance we can explicitly pass in artifacts via a flag rather than magically creating it from environment variables?